### ISSUE TITLE
Crash on Picasso with rich push notifications on Android 8.1.0

### ISSUE THREAD

**Issue by serenasoldo** [Iterable/iterable-android-sdk#26]
When I send a rich push notification on a device with Android O, the app crashes: 

```java
E/AndroidRuntime: FATAL EXCEPTION: main
  Process: com.marleyspoon.staging, PID: 11449
  java.lang.IllegalArgumentException: RemoteViews must not be null.
  at com.squareup.picasso.RequestCreator.into(RequestCreator.java:540)
  at com.iterable.iterableapi.IterableNotification$1.run(IterableNotification.java:56)
  at android.os.Handler.handleCallback(Handler.java:790)
  at android.os.Handler.dispatchMessage(Handler.java:99)
  at android.os.Looper.loop(Looper.java:164)
  at android.app.ActivityThread.main(ActivityThread.java:6494)
  at java.lang.reflect.Method.invoke(Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807)
```

Apparently `notification.bigContentView` can be null (https://developer.android.com/reference/android/app/Notification.html#bigContentView) which must be what happens here. 

The notification is correctly displayed on Android 6.0 and lower.

Any idea on how to fix this?

**Comment by davidtruong** [comment]
@serenasoldo: Taking a look at it.

**Comment by davidtruong** [comment]
@serenasoldo: It looks like the simple fix would be to update the load sequence to fetch the image and set the bigPicture prior to building the notification. I have this fix prioritized for our next SDK release.

**Comment by serenasoldo** [comment]
Thanks a lot @davidtruong 
Do you have any idea about when the new release will be available?

**Comment by davidtruong** [comment]
@serenasoldo: aiming for a release next week. Let me know if you have any other questions.

**Comment by davidtruong** [comment]
In PR now: https://github.com/Iterable/iterable-android-sdk/pull/28. Waiting on final approval. Should have a new release out tomorrow.

**Comment by davidtruong** [comment]
Fixed in Iterable-SDK 2.2.4
